tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -615,6 +614,18 @@ def test_model_monitoring_voting_ensemble(self):
         # it can take ~1 minute for the batch pod to finish running
         sleep(60)
 
+        # Validate that the KV schema has been generated as expected
+        client = get_v3io_client(endpoint=mlrun.mlconf.v3io_api)
+        cursor = client.kv.new_cursor(
+            container="users",
+            table_path=f"pipelines/{self.project_name}/model-endpoints/endpoints/",
+            access_key=os.environ.get("V3IO_ACCESS_KEY"),
+        )
+
+        # List all the files from the KV table path and validate that the schema file is exist
```

Comment:
is exist => exist
also, can we have any assertion on the schema file contents?

Reply:
Actually it should be "exists" :)